src/civis/__init__.py

```diff
@@ -19,18 +20,22 @@ def _lazy_import(name):
     return module
 
 
-futures = _lazy_import("civis.futures")
-io = _lazy_import("civis.io")
-ml = _lazy_import("civis.ml")
-parallel = _lazy_import("civis.parallel")
-utils = _lazy_import("civis.utils")
+if TYPE_CHECKING:
+    from civis import futures, io, ml, parallel, utils
+else:
+    futures = _lazy_import("civis.futures")
+    io = _lazy_import("civis.io")
+    ml = _lazy_import("civis.ml")
+    parallel = _lazy_import("civis.parallel")
+    utils = _lazy_import("civis.utils")
```

Comment:
After #490, one thing I've noticed locally was that my IDE had become blind for type hints and code autocompletes. Fixing this issue by using real imports for tools of static type checks while keeping the lazy imports for actual runtime.